source/compiler/qsc_eval/src/lib.rs

```diff
@@ -161,6 +161,11 @@ pub enum Error {
     #[diagnostic(help("comparing measurement results is not supported when performing circuit synthesis or base profile QIR generation"))]
     ResultComparisonUnsupported(#[label("cannot compare to result")] PackageSpan),
 
+    #[error("cannot compare measurement result from qubit loss")]
+    #[diagnostic(code("Qsc.Eval.ResultLossComparisonUnsupported"))]
+    #[diagnostic(help("use of a measurement result from a qubit that was lost is not supported, use `CheckLoss` to check for loss instead"))]
+    ResultLossComparisonUnsupported(#[label("cannot compare result from qubit loss")] PackageSpan),
```

Comment:
```suggestion
    ResultLossComparisonUnsupported(#[label] PackageSpan),
```

Reply:
Why remove the label? A lot of our errors use them, and it looks nice in the output (though generating this made me realize it should use the whole expression span rather than the right-hand side):
![image](https://github.com/user-attachments/assets/17d215f7-dbe8-402e-b0b3-e4e7c06d9949)